build-system/compile/shorten-license.js

```diff
@@ -64,11 +64,7 @@ const BSD_SHORT = [
 
 const LICENSES = [[MIT_FULL, MIT_SHORT], [POLYMER_BSD_FULL, BSD_SHORT]];
```

Comment:
Can one of these be deleted?

Reply:
Looks like the MIT license can be removed.

Reply:
Done